src/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+  <body>
+ Hello, world!
+  </body>
```

Comment:
it would better look: 
```suggestion
  <body>Hello, world!</body>
```
or 
```suggestion
  <body>
    Hello, world!
  </body>
```

Reply:
@asakevych  Thank you!